deploy/src/main/deploy/lora-adapter.credentials

```diff
@@ -0,0 +1,2 @@
+username=lora-adapter@HONO
+password=lora-secret
```

Comment:
the *lora-adapter@HONO* user also needs to be added to `deploy/example-permissions.json` in order for other services being able to authenticate the LoRa adapter using the Auth Server.

Reply:
Added